includes/external-stories-shortcode.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Handles the registration of the UCF News Shortcode
+ **/
+
+if ( ! class_exists( 'External_Stories_Shortcode' ) ) {
+	class External_Stories_Shortcode {
+		public static function register_shortcode() {
+			add_shortcode( 'ucf-external-stories', array( 'External_Stories_Shortcode', 'sc_external_stories' ) );
+		}
+
+		public static function register_shortcode_interface( $registered_shortcodes ) {
+			if ( class_exists( 'WP_SCIF_Config' ) ) {
+				$fields = array(
+					array(
+						'name'      => 'Title',
+						'param'     => 'title',
+						'desc'      => 'The title to display before the news feed',
+						'type'      => 'text',
+						'default'   => 'In the News'
+					),
+					array(
+						'name'      => 'Layout',
+						'param'     => 'layout',
+						'desc'      => 'The layout to display the news items',
+						'type'      => 'select',
+						'options'   => UCF_News_Config::get_layouts(),
```

Comment:
`UCF_News_Config::get_layouts()` isn't currently designed to return layouts for this shortcode.  Update this method or use a different one for returning the available layouts.

https://github.com/UCF/UCF-News-Plugin/blob/master/includes/ucf-news-config.php#L40-L50

Reply:
Created a new function for pulling layouts.